components/Navbar/Navbar.module.css

```
@@ -94,6 +94,22 @@
+.addEquipmentLink,
+.borrowingsLink {
+  padding: 8px 16px;
+  color: #4a5568;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 600;
+  border-radius: 8px;
+  transition: all 0.2s ease;
+}
+
+:global(.dark) .addEquipmentLink,
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)